src/store/MoviesContext.jsx

```diff
@@ -8,21 +8,28 @@ export const MoviesContext = createContext();
 export const MoviesContextProvider = (props) => {
     const API_URL=`https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}`;
     const SEARCH_API_URL=`https://api.themoviedb.org/3/search/movie?api_key=${API_KEY}`;
+    const MOVIE_API_URL=`https://api.themoviedb.org/3/movie`;
```

Comment:
There is probably a more concise way of doing this, but this is fine. 
As you add more endpoints here you might want to think of a smarter way.